experimental/pgbouncer/pwd_secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name:  pgxl-passwords-collection
+type: Opaque
+data:
+  # You must base64 encode your values. See: https://kubernetes.io/docs/concepts/configuration/secret/
+  pgpass: "eW91cl9wYXNzd29yZDE="
```

Comment:
Please remove specific secret name. You can keep this in a specific branch.

Reply:
ok sure this is just your_password1 which is in the install.sh as an example that people can change to their needs but I understand and will remove it.

Reply:
You are correct. I will do the same and remove it from the other location.